core/src/main/java/io/crate/types/CollectionType.java

```diff
@@ -45,7 +45,7 @@ public CollectionType(DataType<?> innerType) {
     }
 
     /**
-     * Constructor used for the Streamer interface which initializes
+     * Constructor used for the {@link Streamer} interface which initializes
```

Comment:
Oh yes, of course it is `Streamable` and not `Streamer`. We implement both here but the ctor affects `Streamable`.